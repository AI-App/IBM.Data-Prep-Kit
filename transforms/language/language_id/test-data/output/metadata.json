{
    "pipeline": "pipeline_id",
    "job details": {
        "job category": "preprocessing",
        "job name": "LangIdentification",
        "job type": "ray",
        "job id": "job_id",
        "start_time": "2024-04-05 15:43:51",
        "end_time": "2024-04-05 15:43:54",
        "status": "success"
    },
    "code": {
        "github": "github",
        "commit_hash": "12345",
        "path": "path"
    },
    "job_input_params": {
        "lang_id_model_kind": "fasttext",
        "lang_id_model_url": "facebook/fasttext-language-identification",
        "lang_id_content_column_name": "text",
        "checkpointing": false,
        "max_files": -1,
        "random_samples": -1,
        "files_to_use": [".parquet"],
        "number of workers": 2,
        "worker options": {
            "num_cpus": 1
        },
        "actor creation delay": 0
    },
    "execution_stats": {
        "cpus": 10,
        "gpus": 0,
        "memory": 13.89394836500287,
        "object_store": 2.0
    },
    "job_output_stats": {
        "source_files": 3,
        "source_size": 977745,
        "result_files": 3,
        "result_size": 9123,
        "table_processing": 0.4174013137817383,
        "en": 357,
        "de": 38,
        "vi": 16,
        "ko-Hang": 49,
        "lb": 2,
        "ca": 5,
        "rm": 2,
        "lt": 1,
        "yue-Hant": 58,
        "hu": 1,
        "sv": 3,
        "it": 2,
        "vec-Latn": 2,
        "azb-Arab": 7,
        "tr": 1,
        "fr": 4,
        "ro": 4,
        "pl": 12,
        "cs": 7,
        "es": 3,
        "ast-Latn": 4,
        "eo": 2,
        "oc-Latn": 3,
        "lmo-Latn": 1,
        "da": 1,
        "eu": 1,
        "nl": 4,
        "sk": 1,
        "lvs-Latn": 1,
        "li-Latn": 1,
        "nn": 1,
        "bo-Tibt": 4,
        "af": 1,
        "nb": 1
    },
    "source": {
        "name": "/Users/dawood/git/fm-data-engineering/transforms/language/language_id/test-data/input",
        "type": "path"
    },
    "target": {
        "name": "/Users/dawood/git/fm-data-engineering/transforms/language/language_id/test-data/output",
        "type": "path"
    }
}
