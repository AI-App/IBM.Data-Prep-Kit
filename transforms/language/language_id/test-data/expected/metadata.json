{
    "pipeline": "pipeline_id",
    "job details": {
        "job category": "preprocessing",
        "job name": "LangIdentification",
        "job type": "ray",
        "job id": "job_id",
        "start_time": "2024-03-11 11:14:06",
        "end_time": "2024-03-11 11:15:04",
        "status": "success"
    },
    "code": {
        "github": "github",
        "commit_hash": "12345",
        "path": "path"
    },
    "job_input_params": {
        "drop_column_if_existed": true,
        "model_kind": "fasttext",
        "model_url": "facebook/fasttext-language-identification",
        "content_column_name": "text",
        "checkpointing": false,
        "max_files": -1,
        "number of workers": 2,
        "worker options": {
            "num_cpus": 1
        },
        "actor creation delay": 0
    },
    "execution_stats": {
        "cpus": 4,
        "gpus": 0,
        "memory": 4.5122314458712935,
        "object_store": 2.2561157224699855
    },
    "job_output_stats": {
        "source_files": 3,
        "source_size": 977745,
        "result_files": 3,
        "result_size": 9123,
        "table_processing": 0.7192928791046143,
        "de": 38,
        "pl": 12,
        "cs": 7,
        "sk": 1,
        "en": 357,
        "lvs-Latn": 1,
        "azb-Arab": 7,
        "vi": 16,
        "fr": 4,
        "li-Latn": 1,
        "ko-Hang": 49,
        "nn": 1,
        "es": 3,
        "nl": 4,
        "sv": 3,
        "rm": 2,
        "lb": 2,
        "yue-Hant": 58,
        "bo-Tibt": 4,
        "af": 1,
        "oc-Latn": 3,
        "nb": 1,
        "ca": 5,
        "lt": 1,
        "hu": 1,
        "it": 2,
        "vec-Latn": 2,
        "tr": 1,
        "ro": 4,
        "ast-Latn": 4,
        "eo": 2,
        "lmo-Latn": 1,
        "da": 1,
        "eu": 1
    },
    "source": {
        "name": "/test-data",
        "type": "path"
    },
    "target": {
        "name": "/test-data/output",
        "type": "path"
    }
}
